data/eurucamp/eurucamp-2015/videos.yml

```diff
@@ -1,6 +1,402 @@
+  description: |-
+    The Struggle to Stay Technical by Kinsey Ann Durham
+
+    Life is good. More and more people who never thought they could code, people of diverse backgrounds are learning to write Ruby and become full time developers. Outreach programs are doing an amazing job of getting these people into the community, but now we are facing an even bigger problem. Getting them to stay there. 57% of women alone leave the tech industry once they are there. At every turn, we are told we can’t do it, offered different untechnical positions, or bullied out by toxic cultures. This talk will explore the barriers people of different backgrounds face to stay in technical positions, even once they have overcome the barriers to get there. It will present different perspectives and bring light to things we can do as an individual, an employer, a co-worker and as a community to solve this large problem that we face.
+  video_provider: "youtube"
+  video_id: "D5svAP1aPCk"
+
+- title: "The Power Of Small Abstractions"
+  raw_title: 'eurucamp 2015 - The Power Of Small Abstractions by Josep M. "Tsux" Bach'
+  speakers:
+    - Josep M. "Tsux" Bach
+  date: "2015-08-02"
+  event_name: "eurucamp 2015"
+  published_at: "2015-08-14"
+  announced_at: "2014-11-17"
+  description: |-
+    The Power Of Small Abstractions by Josep M. "Tsux" Bach
+
+    Every time we solve an everyday programming problem we learn from its solution. When we come across a similar problem later on, we think “aha! I’ve seen this before! I know how to solve it!”. Many of us are also familiar with design patterns, which are abstractions that solve entire classes of problems. There also exists, however, a different kind of pattern. But, as opposed to help you structure a whole compiler or business application, these patterns love hiding in small things like methods or functions, or even binary operations. They whisper to you when you concatenate two strings. They leave a trail of breadcrumbs every time you map over a list. These abstractions have superpowers, too. They can separate the what from the when, or even from the how. They can add together things other than numbers. They can control time, or take over the control flow of your program entirely. In this talk you will discover the amazing power of small abstractions. After that you’ll start hearing their whispers and seeing their breadcrumb trails. And only then you will be ready to let them unfold their full potential in your programs.
+  video_provider: "youtube"
+  video_id: "U_yIxD_U-IY"
+
+- title: "Lightning Talks"
+  raw_title: "eurucamp 2015 - Lightning Talks"
+  speakers:
+    - "Isaiah Peng"
+    - Ruby Corns study group
+  date: "2015-08-02"
+  event_name: "eurucamp 2015"
+  published_at: "2015-08-14"
+  announced_at: "2014-11-17"
+  description: |-
+    Protocol buffers on JRuby by Isaiah Peng
+
+    Study Groups by Ruby Corns
+
+    Ruby Issues
+
+    Shoes
+  video_provider: "youtube"
+  video_id: "mJcxAY172y8"
```

Comment:
Would you mind adding these 4 lightning talks as subtalks for the "Lightning Talks" video? Thank you!